docs/changelog.md

```diff
@@ -11,3 +11,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Installation and run scripts
 - Changelog
+- Docker support
+
+### Fixed
+
+- HTML tags in sc-links
```

Comment:
![image](https://github.com/ostis-ai/react-sc-web/assets/36566977/9b074103-87b1-43ad-943b-43eab3f547da)
![image](https://github.com/ostis-ai/react-sc-web/assets/36566977/091cb989-44c7-4810-b401-19ddc4925716)

Reply:
Looks like this feature has some buggy behavior in general, because some of the tags work just fine. Maybe this is a bug in `sc-web` (or maybe even expected behavior)
![image](https://github.com/ostis-ai/react-sc-web/assets/73379561/8a3edb13-ee6b-46ee-afa4-06ad5fc427b3)